cognite/neat/_utils/validation.py

```diff
@@ -84,10 +84,7 @@ def humanize_validation_error(
     elif type_ == "literal_error":
         msg = f"{error['msg']}. Got {error['input']!r}."
     elif type_ == "string_type":
-        msg = (
-            f"{error['msg']}. Got {error['input']!r} of type {type(error['input']).__name__}. "
-            f"Hint: Use double quotes to force string."
-        )
+        msg = f"{error['msg']}. Got {error['input']!r} of type {type(error['input']).__name__}. "
```

Comment:
This is a good one. It comes from Toolkit were this made a lot of sense. But it doesn't for neat